chapters/hpmor-chapter-002.tex

```diff
@@ -29,7 +29,7 @@ \chapter{Everything I Believe Is False}
 
 His father was lowered carefully to the ground.
 
-Harry ruffled a hand through his own hair. Maybe it was just that strange part of him which had \emph{already} been convinced, but…“That’s a bit of an anticlimax,” Harry said. “You’d think there’d be some kind of more dramatic mental event associated with updating on an observation of infinitesimal probability—” Harry stopped himself. Mum, the witch, and even his Dad were giving him \emph{that look} again. “I mean, with finding out that everything I believe is false.”
+Harry ruffled a hand through his own hair. Maybe it was just that strange part of him which had \emph{already} been convinced, but… “That’s a bit of an anticlimax,” Harry said. “You’d think there’d be some kind of more dramatic mental event associated with updating on an observation of infinitesimal probability—” Harry stopped himself. Mum, the witch, and even his Dad were giving him \emph{that look} again. “I mean, with finding out that everything I believe is false.”
```

Comment:
I think this was correct without the space: it's not the end of a sentence (which would require a full stop after the ellipsis), even though the speech starts with a new sentence.